lib/report.c

```diff
@@ -65,7 +65,7 @@ grn_report_table(grn_ctx *ctx,
                  grn_obj *table)
 {
   grn_obj description;
-  grn_obj *target;
+  grn_obj *target, *previous_target = NULL;
```

Comment:
Could you declare one variable per line?

Reply:
I have fixed it.